en/_posts/2017-10-10-pgroonga-2.0.2.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: post.en
-title: PGroonga (fast full text search module for PostgreSQL) 2.0.3 has been released
-description: PGroonga (fast full text search module for PostgreSQL) 2.0.3 has been released!
+title: PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released
+description: PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released!
 ---
 
-## PGroonga (fast full text search module for PostgreSQL) 2.0.3 has been released
+## PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released
 
-[PGroonga](https://pgroonga.github.io/) 2.0.3 has been released! PGroonga makes PostgreSQL fast full text search for all languages.This is major version up!
+[PGroonga](https://pgroonga.github.io/) 2.0.2 has been released! PGroonga makes PostgreSQL fast full text search for all languages.This is major version up!
```

Comment:
We need missing space after `.` ?

Reply:
(Question) The previous PGroonga' version is 2.0.1, so isn't 2.0.2 major version ? 🤔 (Also in Japanese)

Reply:
No public release announcement for 2.0.0 and 2.0.1 here, then 2.0.2 is the first announcement for 2.X major versin up.

Reply:
I see. I wondered If this sentence confused users with 2.0.0 or 2.0.1, but you don't worry about it, I think it's OK. 😃